cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -18,6 +18,20 @@ Existing tools such as `oc` are more operations-focused and require a deep under
 * Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the {product-title} cluster for deployment.
 * Official full support of Node.js and Java components.
-* Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
+* Partial compatibility with languages and frameworks such as Ruby, Perl, PHP, and Python. 
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the {product-title} cluster.
\ No newline at end of file
+* Lists all the available components and services from the {product-title} cluster.
+
+=== Supported languages and corresponding container images
+Run `odo catalog list components` to list the supported languages and corresponding container images present on your cluster.
+
```

Comment:
Is there anything else to say about the partially-compatible components in this section? Are they not listed in the catalog?

Reply:
No. `odo catalog list components` lists only the components supported as of now. @kadel linking you, just in case I misunderstood something. Thanks!

Reply:
`odo catalog list components` list both, supported and unsupported, each section is clearly marked in the output
Here is an example of the output:
```
▶ odo catalog list components
Odo Supported OpenShift Components:
NAME       PROJECT       TAGS
java       openshift     8,latest
nodejs     openshift     10,8,8-RHOAR,latest

Odo Unsupported OpenShift Components:
NAME                      PROJECT       TAGS
dotnet                    openshift     1.0,1.1,2.1,2.2,latest
fuse7-eap-openshift       openshift     1.3
fuse7-java-openshift      openshift     1.3
fuse7-karaf-openshift     openshift     1.3
httpd                     openshift     2.4,latest
nginx                     openshift     1.10,1.12,1.8,latest
nodejs                    openshift     6
perl                      openshift     5.24,5.26,latest
php                       openshift     7.0,7.1,latest
python                    openshift     2.7,3.5,3.6,latest
```

Reply:
I did not know this, fixing the docs. Thanks!

Reply:
@pmacko1 we need to ensure that we test the commands before we document them. Also, as discussed, as per OCP guidelines for aligned products, please ensure that we have SME, QE, and peer reviews done by our team before we ask for reviews from the OCP team. I realize this is your first PR for odo, but hence forth request you to keep these in mind. Thank you.